openhands/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -10,9 +10,17 @@
     module=file_reader, function_names=file_reader.__all__, target_globals=globals()
 )
 __all__ = file_ops.__all__ + file_reader.__all__
+__except__ = [
+    'edit_file_by_replace',
+    'insert_content_at_line',
+    'append_file',
+]  ## DISABLED TEMPORARILY.
```

Comment:
You might also consider disable things like `create_file` and include some instruction in the prompt, telling the LLM that you can also create new files using `EditAction` (if this is true ofc)

Reply:
Thats a good idea, maybe we can do that in a subsequent PR?

Reply:
Yeah - i think you can experiment with this when you are running benchmarks (e.g., try this and see if it increases score)